include/core/mimedata/mimedata.hpp

```diff
@@ -74,6 +74,11 @@ namespace Toolbox {
             return *this;
         }
 
+        static bool isMimeTarget(std::string target) {
```

Comment:
Pass by const reference.

Reply:
Fixed by [this commit](https://github.com/JoshuaMKW/JuniorsToolbox/pull/36/commits/565d229c848d5514ee703fd7ec29eb76885dcaa1)